bika/lims/browser/worksheet/views/results.py

```diff
@@ -99,11 +105,12 @@ def __call__(self):
                         attachments.append(other.UID())
                     attachments.append(attachment.UID())
                     analysis.setAttachment(attachments)
+        # /TODO
```

Comment:
```
analysis.reindexObject()
```
is also required here, when option "All Analyses of Service" is used (`service_uid`, but not `analysis_uid`)